package/windows/pyinstaller.spec

```diff
@@ -9,47 +13,131 @@ datas.append((os.path.join(PROJECT_PATH, "README.rst"), "."))
 datas.append((os.path.join(PROJECT_PATH, "LICENSE"), "."))
 datas.append((os.path.join(PROJECT_PATH, "copyright"), "."))
 datas += collect_data_files("silx.resources")
+datas += collect_data_files("hdf5plugin")
 
 
-hiddenimports = collect_submodules('fabio')
+hiddenimports = []
+hiddenimports += collect_submodules('fabio')
+hiddenimports += collect_submodules('hdf5plugin')

```

Comment:
@vallsv `hdf5plugin` is added here

Reply:
Nice. Thx.

Reply:
Should `hdf5plugin` be added to the requirements-dev.txt?

Reply:
Yes, indeed!